src/OpenTelemetry.Api/Metrics/BlankLabelSet.cs

```diff
@@ -0,0 +1,29 @@
+﻿// <copyright file="BlankLabelSet.cs" company="OpenTelemetry Authors">
+// Copyright 2018, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System.Collections.Generic;
+
+namespace OpenTelemetry.Metrics
+{
+    /// <summary>
+    /// Blank or no-op labelset.
+    /// </summary>
+    public sealed class BlankLabelSet : LabelSet
+    {
+        /// <inheritdoc/>
+        public override IEnumerable<KeyValuePair<string, string>> Labels { get; set; }
```

Comment:
This needs to override? The abstract class only has a getter and setter too

Reply:
@bruno-garcia Took your suggestion and made abstract class provide Empty implementation for the same.

Reply:
Thanks @cijothomas !